vars/npmExecuteEndToEndTests.groovy

```diff
@@ -0,0 +1,137 @@
+import com.sap.piper.ConfigurationHelper
+import com.sap.piper.DownloadCacheUtils
+import com.sap.piper.GenerateDocumentation
+import com.sap.piper.k8s.ContainerMap
+import groovy.transform.Field
+import com.sap.piper.Utils
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field String STEP_NAME = getClass().getName()
+
+@Field Set GENERAL_CONFIG_KEYS = [
+    /** Executes the deployments in parallel.*/
+    'parallelExecution'
```

Comment:
should probably also have this alias? https://github.com/SAP/jenkins-library/blob/master/vars/multicloudDeploy.groovy#L45